snmp/datadog_checks/snmp/data/default_profiles/extreme-switching.yaml

```diff
@@ -0,0 +1,123 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-entity-sensor.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.1916.2.89
+  - 1.3.6.1.4.1.1916.2.343
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "extreme-networks"
+
+metrics:
+  - MIB: EXTREME-SOFTWARE-MONITOR-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.1916.1.32.1.2.0
+  # - MIB: EXTREME-SOFTWARE-MONITOR-MIB
+  #   table:
+  #     name: extremeMemoryMonitorSystemTable
+  #     OID: 1.3.6.1.4.1.1916.1.32.2.2
+  #   symbols:
+  #     - name: memory.total
+  #       OID: 1.3.6.1.4.1.1916.1.32.2.2.1.2
+  #     - name: memory.free
+  #       OID: 1.3.6.1.4.1.1916.1.32.2.2.1.3
+  #   metric_tags:
+  #     - column:
+  #         OID: 1.3.6.1.4.1.1916.1.32.2.2.1.1
+  #         name: extremeMemoryMonitorSystemSlotId
+  #       tag: extreme_memory_monitor_system_slot_id
+  #     TODO: enum in scalar metric is not supported yet (keep this metric and this
+  #       comment in profile until it's fixed)
```

Comment:
Q: Why are we ignoring memory metrics?

Is this generated:
```
  #     TODO: enum in scalar metric is not supported yet (keep this metric and this
  #       comment in profile until it's fixed)
```
In this case, we have table metrics, I'm wondering why we have this TODO.

Reply:
the memory metrics here are of type string

Reply:
Indeed, a bit strange.

That said, type is DisplayString, but I think the value is still a integer (as string).

```
        extremeMemoryMonitorSystemTotal OBJECT-TYPE
            SYNTAX DisplayString (SIZE(0..8))
            MAX-ACCESS read-only
            STATUS current
            DESCRIPTION
                  "Total amount of DRAM in Kbytes in the system."
```

Example: https://kb.paessler.com/en/topic/60231-monitor-extreme-networks-switch-x450e-48p-cpu-and-mem

```
R5-460-btm.10 # show snmp get-next 1.3.6.1.4.1.1916.1.32.2.2.1.2 extremeSwMonitor.2.2.1.2.1 = 1048576
R5-460-btm.11 # show snmp get-next 1.3.6.1.4.1.1916.1.32.2.2.1.4 extremeSwMonitor.2.2.1.4.1 = 27052
R5-460-btm.12 # show snmp get-next 1.3.6.1.4.1.1916.1.32.2.2.1.5 extremeSwMonitor.2.2.1.5.1 = 217220
R5-460-btm.13 # show snmp get-next 1.3.6.1.4.1.1916.1.32.2.2.1.3 extremeSwMonitor.2.3.1.1.1.97.97.97 = 804304
```

Reply:
do we have a way of handling this now ?
I don't know if adding `metric_type` would do a conversion ?